e2e/vue3-vite-e2e/tests/component.spec.ts

```diff
@@ -24,53 +24,53 @@ describe('component e2e', () => {
     // Check file exists
     const componentName = names(component).className;
     checkFilesExist(
-      `apps/${app}/src/components/${componentName}.cy.ts`,
-      `apps/${app}/src/components/${componentName}.vue`
+      `apps/${app}/src/app/components/${componentName}.cy.ts`,
+      `apps/${app}/src/app/components/${componentName}.vue`
     );
   });
 
   describe('--project', () => {
     it('should create component in the specified project', async () => {
```

Comment:
I modify this test case a little bit to see if the `--project` option works for both app and library projects.
Originally we create two apps, now we create 1 app and 1 library.